deepfence_frontend/package.json

```diff
@@ -22,5 +22,8 @@
   "engines": {
     "node": ">=18.0.0"
   },
-  "packageManager": "pnpm@7.17.1"
+  "packageManager": "pnpm@7.17.1",
+  "dependencies": {
+    "moment": "^2.29.4"
```

Comment:
dependencies should not be added here, they should be added to respective packages. We are avoiding moment package, there is already an alternative package installed in dashboard package.